specification/recoveryservicessiterecovery/resource-manager/readme.md

```diff
@@ -33,6 +33,11 @@ directive:
       - $.paths
     suppress:
       - UniqueResourcePaths
+
+  - from: service.json
```

Comment:
This should defined in individual suppression section like below.
```
## Suppression

``` yaml
directive:
  - from: service.json
    suppress: OAV131 
    reason: Testing purpose.
```

Reply:
added separate section